src/controls/MenuSelectTableBorderStyle.tsx

```diff
@@ -0,0 +1,193 @@
+import BorderStyle from "@mui/icons-material/BorderStyle";
+import { ClickAwayListener, Fade, Paper, Popper } from "@mui/material";
+import * as React from "react";
+import { makeStyles } from "tss-react/mui";
+import { useRichTextEditorContext } from "../context";
+import {
+  BorderStyleDashed,
+  BorderStyleDotted,
+  BorderStyleDouble,
+  BorderStyleGroove,
+  BorderStyleInset,
+  BorderStyleNone,
+  BorderStyleOutset,
+  BorderStyleRidge,
+  BorderStyleSolid,
+} from "../icons";
+import { Z_INDEXES } from "../styles";
+import MenuButton from "./MenuButton";
+import type { MenuButtonTooltipProps } from "./MenuButtonTooltip";
+import MenuControlsContainer from "./MenuControlsContainer";
+
+export type BorderStyleSelectOption = {
+  value: string;
+  IconComponent: React.ElementType<{
+    className: string;
+  }>;
+  label?: string;
+  shortcutKeys?: MenuButtonTooltipProps["shortcutKeys"];
+};
+
+export type MenuSelectTableBorderStyleProps = {
+  options?: BorderStyleSelectOption[];
+  label?: string;
+  className?: string;
+};
+
+const useStyles = makeStyles({ name: { MenuSelectTableBorderStyle } })(
+  (theme) => ({
+    root: {
+      zIndex: Z_INDEXES.BUBBLE_MENU,
+    },
+
+    paper: {
+      backgroundColor: theme.palette.background.default,
+    },
+  })
+);
+
+const DEFAULT_BORDER_STYLE_OPTIONS: BorderStyleSelectOption[] = [
```

Comment:
Is it correct that many of these options don't display differently than "solid" with the default mui-tiptap styles that have borderWidth set to 1px, and they'd require >1px borders to be visually distinct?